hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StripeStoreFileManager.java

```diff
@@ -56,7 +56,7 @@
  * When L0 is compacted, it's split into the files corresponding to existing stripe boundaries,
  * that can thus be added to stripes.
  * When scan or get happens, it only has to read the files from the corresponding stripes.
- * See StripeCompationPolicy on how the stripes are determined; this class doesn't care.
+ * See StripeCompactionPolicy on how the stripes are determined; this class doesn't care.
```

Comment:
Also add a Javadoc link to the class?

Reply:
@HorizonNet, I made a mistake and this pull request was closed and a [new pull request](https://github.com/apache/hbase/pull/2653) is opened.

Reply:
I've done your comment in that pull request.